pig/src/main/java/org/apache/zeppelin/pig/PigInterpreter.java

```diff
@@ -58,6 +59,12 @@ public void open() {
     }
     try {
       pigServer = new PigServer(execType);
+      for (Map.Entry entry : getProperty().entrySet()) {
+        if (!entry.getKey().toString().startsWith("zeppelin.")) {
+          pigServer.getPigContext().getProperties().setProperty(entry.getKey().toString(),
+              entry.getValue().toString());
```

Comment:
There's 2 types of property for zeppelin interpreter. Type-1 is used by interpreter itself, Type-2 is used by the engine underneath (like spark engine). All spark configuration start with `spark.`, this is just spark's convention, not zeppelin's convention. zeppelin's convention is to use `zeppelin` as the prefix for Type-1 property, and keep Type-2 property as what it is in the underlying engine. So here I keep the pig property as what it is in pig, otherwise it would confuse users IMO.